L/louvain/build_tarballs.jl

```diff
@@ -13,10 +13,9 @@ sources = [
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/louvain/
-make -j${nproc}
-mkdir -p ${bindir}
+make -j${nproc} CXX="${CXX}"
```

Comment:
The redundant `CXX="${CXX}"` isn't needed if you update to the latest revision now.

Reply:
True, I switched the source version and removed the change with CXX assignment.